src/Neo.ConsoleService/System/IsExternalInit.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2015-2024 The Neo Project.
```

Comment:
What is this file, I did not understand who uses it.

Reply:
`dotnet` only support api calls of the OS. its required for compiler to compile the `class` that have `init` for properties